samples/addons/kiali.yaml

```diff
@@ -49,7 +49,7 @@ data:
       custom_secrets: []
       host_aliases: []
       hpa:
-        api_version: autoscaling/v2beta2
+        api_version: autoscaling/v2
```

Comment:
should update the source (manifests/addons/values-kiali.yaml)

Reply:
will do